### PR TITLE
Readme update to 1.2.0

### PR DESCRIPTION
**Pull request by aaronrothschild**
- Added new section for instructions for 1.20
- Added a configuration example and screenshots
- Moved pre-1.10 instructions to bottom of page for users who still need a reference to config.json format for now.  Can deprecate later.

Address #30

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,14 +5,46 @@
...
+ - Search for "Custom Attribute" or manually find the plugin from the list and click **Install**
+ - After the plugin has downloaded and been installed, click the **Configure** button
+ - Set **Enable Plugin** to **True**, then click "Save" at the bottom of the screen
+ - Proceed to Configure your Attribute rules
+ 
+ ## Configuration
+ 
+Before you start, Identify the attributes you want to display on a user's profile popover.  These can contain emojis. Some examples could be "Timezone:PST", "Development Team", "Executive Team Member", "Mentor", etc. then Identify the groups or particular usernames that should display those atrributes.  A spreadsheet can help to organize things.
+
+1. Click "Add Custom Attribute" button, a text box will appear.  Add text and/or emojis to the field.  This text will be displayed on a user's profile.
```
Comment (marianunez) [inline README.md:31]: The attribute text actually support markdown so it could also be a link. So maybe we can rephrase it to something like (only a suggestions, reword as needed) 
```suggestion
1. Click "Add Custom Attribute" button, a text box will appear.  Add the text that would appear in the user's profile popover. The text supports markdown and could include emojis and/or links, i.e. "[Integrations Team](https://developers.mattermost.com/internal/rd-teams/#integrations-team)"
```

**Comment by aaronrothschild** [comment]
@larkox Can we merge this PR into master?
